getting-started/installation-ubuntu-ami.md

```diff
@@ -0,0 +1,36 @@
+## Installing from an Amazon AMI (Ubuntu) [](installation-ubuntu-ami)
+
+TimescaleDB is currently available as an Ubuntu 16.04 Amazon EBS-backed AMI. AMIs are
+distributed by region, and our AMI is currently available in `us-east-1`, `us-east-2`,
+`us-west-1`, and `us-west-2`. Note that this image is built to use an EBS attached volume
+rather than the default disk that comes with EC2 instances.
+
+See below for the image id corresponding to each region for the most recent TimescaleDB version:
+
+Region | Image ID
+--- | ---
+us-east-1 (North Virginia) | ami-0ecf0cc105d5a8fc0
+us-east-2 (Ohio) | ami-0bd0ceaa391794533
+us-west-1 (North California) | ami-03645fd772b136a5a
+us-west-2 (Oregon) | ami-0566aa080793cac7c
+
+
+To launch the AMI, go to the `AMIs` section of your AWS EC2 Dashboard and select
```

Comment:
Just for readability, can we make these bullets?

Reply:
They are in a table right now.  Is that not easy to read?

Reply:
They are in a table right now.  Is that not easy to read?